flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendSavepointTest.java

```diff
@@ -200,7 +196,40 @@ public void testTriggerSavepointCustomFormat() throws Exception {
             MockedCliFrontend frontend = new MockedCliFrontend(clusterClient);
 
             String[] parameters = {
-                jobId.toString(), savepointDirectory, "-type", SavepointFormatType.NATIVE.toString()
+                jobId.toString(), savepointDirectory, "-t", SavepointFormatType.NATIVE.toString()
+            };
+            frontend.savepoint(parameters);
+
+            verify(clusterClient, times(1))
+                    .triggerSavepoint(
+                            eq(jobId), eq(savepointDirectory), eq(SavepointFormatType.NATIVE));
+
+            assertTrue(buffer.toString().contains(savepointDirectory));
+        } finally {
+            clusterClient.close();
+
+            restoreStdOutAndStdErr();
+        }
+    }
+
+    @Test
+    public void testTriggerSavepointCustomFormatLongOption() throws Exception {
```

Comment:
I think the only difference of these two test methods is the option, one is `-t` and another is `--type`. We could use one base private method which contains the main body and two test case with different option key to reduce code duplication.